data/campaigns/Winds_of_Fate/scenarios/08_Overlook.cfg

```diff
@@ -895,7 +895,7 @@ Just like we did the first one."
             [filter]
                 id=Resha
             [/filter]
-            variable=Resha
+            variable=resha
```

Comment:
Wondering if this is a much more severe bug than simply a wmllint warning. Without this fix, does Resha spawn in S10 with no experience?

Reply:
She will, but respawning as a L3 AMLA, that issue in particular is not too bad.

However, if the player gave her the magical cold attack in S7 then it will be lost, so for that reason I definitely want to backport this fix to 1.18.